.bin/create-rust-webpack.js

```diff
@@ -6,9 +6,11 @@ const clone = spawn("git", ["clone", "https://github.com/rustwasm/rust-webpack-t
 
 clone.on("close", (code) => {
   if (code !== 0) {
+    // TODO(sven): handle error here
     console.error()
     process.exit(code);
   } else {
     console.log("Rust + WebAssembly + Webpack = <3");
+    // TODO(sven): npm install
```

Comment:
Instead of TODO comments, can we file issues?

Unless you were intending to fix these TODOs before we merge this.

Reply:
https://github.com/rustwasm/rust-webpack-template/issues/12